apps/medical-illustration/webcomponent-example.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Medical Illustration Web Component</title>
+    <base href="/" />
```

Comment:
Does this web component rely on base href? It really shouldn't. Needs to be able to work flexibly and standalone. I think it's true here, but just want to check.

Reply:
I don't think that needs to be there, will remove it.